R/config.R

```diff
@@ -1,10 +1,23 @@
 
 
 rsconnectConfigDir <- function(subDir = NULL) {
-  applicationConfigDir("connect", subDir)
+
+  # first check whether the main rsconnect directory exists
```

Comment:
I don't think that will be a concern (btw the fact that the application config dir was named "connect" wasn't intentional and is pretty close to being a typo). The logical name for both directories is "rsconnect" and hopefully "RootPath" connotes a primary data storage location and "ConfigDir" connotes configuration data.